experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -0,0 +1,123 @@
+## helpers for user-specified IO
+include("debug_plots.jl")
+include("diagnostics_plots.jl")
+
+"""
+    postprocess_sim(mode_type::AbstractSlabPlanetModeType, cs, postprocessing_vars)
+
+If they exist, perform conservation checks, for any slabplanet simulation, and then call
+`common_postprocessing` to perform common postprocessing tasks that are common to all simulation types.
+"""
+function postprocess_sim(mode_type::AbstractSlabPlanetModeType, cs, postprocessing_vars)
+    (;
+        dir_paths,
+        plot_diagnostics,
+        use_coupler_diagnostics,
+        output_default_diagnostics,
+        t_end,
+        conservation_softfail,
+        atmos_output_dir,
+        amip_diags_handler,
+        comms_ctx,
+    ) = postprocessing_vars
```

Comment:
`dir_paths`, `amip_diags_handler`, and `comms_ctx` are in `cs` and can be accessed from there in this function